services/web/server/src/simcore_service_webserver/security/_authz_db.py

```diff
@@ -0,0 +1,45 @@
+import logging
+from typing import TypedDict
+
+import sqlalchemy as sa
+from aiopg.sa import Engine
+from aiopg.sa.result import ResultProxy
+from models_library.basic_types import IdInt
+from pydantic import parse_obj_as
+from servicelib.aiohttp.aiopg_utils import PostgresRetryPolicyUponOperation
+from simcore_postgres_database.models.users import UserRole
+from tenacity import retry
+
+from ..db.models import UserStatus, users
+from ._identity import IdentityStr
+
+_logger = logging.getLogger(__name__)
+
+
+class UserInfoDict(TypedDict, total=True):
+    id: IdInt
+    role: UserRole
+
+
+@retry(**PostgresRetryPolicyUponOperation(_logger).kwargs)
```

Comment:
question: why do we have this retry here?
I think we do not use this very consistently. and also I wonder if sqlalchemy does not have some built ins... also... I wonder a bit about the significance of losing postgres connectivity.

Reply:
Again this is legacy from the times we got disconnections from the database (remember when we run out of clients ;-) !). I guess at this point we can drop it as well.